public/file.json

```diff
@@ -1,339 +1,4 @@
-            "download": 34.042,
-            "upload": 14.904
-        },
-        {
-            "date": "2020-7-8 1:12",
-            "download": 35.733,
-            "upload": 14.918
-        },
-        {
-            "date": "2020-7-8 1:14",
-            "download": 49.092,
-            "upload": 14.963
-        },
-        {
-            "date": "2020-7-8 1:16",
-            "download": 43.7,
-            "upload": 15.3
-        },
-        {
-            "date": "2020-7-8 1:18",
-            "download": 31.326,
-            "upload": 15.643
-        },
-        {
-            "date": "2020-7-8 1:20",
-            "download": 27.7,
-            "upload": 14.781
-        },
-        {
-            "date": "2020-7-8 1:22",
-            "download": 33.431,
-            "upload": 15.184
-        },
-        {
-            "date": "2020-7-8 1:24",
-            "download": 31.451,
-            "upload": 14.594
-        },
-        {
-            "date": "2020-7-8 1:26",
-            "download": 32.829,
-            "upload": 15.44
-        },
-        {
-            "date": "2020-7-8 1:28",
-            "download": 32.58,
-            "upload": 15.854
-        },
-        {
-            "date": "2020-7-8 1:30",
-            "download": 21.948,
-            "upload": 12.006
-        },
-        {
-            "date": "2020-7-8 1:32",
-            "download": 35.39,
-            "upload": 14.757
-        },
-        {
-            "date": "2020-7-8 1:34",
-            "download": 34.242,
-            "upload": 15.698
-        },
-        {
-            "date": "2020-7-8 1:36",
-            "download": 33.844,
-            "upload": 15.539
-        },
-        {
-            "date": "2020-7-8 1:38",
-            "download": 34.497,
-            "upload": 14.895
-        },
-        {
-            "date": "2020-7-8 1:40",
-            "download": 46.194,
-            "upload": 15.449
-        },
-        {
-            "date": "2020-7-8 1:42",
-            "download": 33.991,
-            "upload": 14.54
-        },
-        {
-            "date": "2020-7-8 1:44",
-            "download": 34.139,
-            "upload": 15.005
-        },
-        {
-            "date": "2020-7-8 1:46",
-            "download": 29.909,
-            "upload": 16.743
-        },
-        {
-            "date": "2020-7-8 1:48",
-            "download": 41.549,
-            "upload": 15.93
-        },
-        {
-            "date": "2020-7-8 1:50",
-            "download": 32.859,
-            "upload": 14.933
-        },
-        {
-            "date": "2020-7-8 1:52",
-            "download": 29.18,
-            "upload": 14.972
-        },
-        {
-            "date": "2020-7-8 1:54",
-            "download": 32.364,
-            "upload": 14.972
-        },
-        {
-            "date": "2020-7-8 1:56",
-            "download": 33.586,
-            "upload": 15.389
-        }
-    ]
+    "Speedtests": [],
+    "Availability": []
```

Comment:
I would actually suggest gitignoring the output file, since it will immediately get filled up with personal data that we probably won't want to ever commit. I realise this may mean removing the file and starting a new output file- I can make a PR for this soon if you can't get to it 👍

Reply:
Yeah I agree, no need to keep it around :)